### PR TITLE
Fixed unstable tests on ProcessUtils

### PR DESCRIPTION
**Pull request by dutor**
As title.

### CODE REVIEW

File: src/common/process/test/ProcessTest.cpp [hunk 1]
```diff
@@ -43,12 +43,15 @@ TEST(ProcessUtils, isPidAvailable) {
     }
     {
         // pid file which contains pid of current process
-        auto pidFile = "/tmp/maybe-non-existing-dir/process_test.pid";
+        auto pidFile = folly::stringPrintf("/tmp/non-existing-dir-%d/process_test.pid", ::getpid());
```
Comment (dangleptr) [inline src/common/process/test/ProcessTest.cpp]: Why not TempDir?

Comment (dutor) [reply src/common/process/test/ProcessTest.cpp]: I had considered it.

But TempDir for TempFile will create the directory or file and be capable of deleting them automatically. But what I need here is a temp name and auto deletion, but not the file creation.

**Comment by dutor** [comment]
> Unit testing passed.

What the hell.

File: src/common/process/test/ProcessTest.cpp (hunk 1 above)
Comment (dangleptr) [reply src/common/process/test/ProcessTest.cpp]: Fair enough

**Pull request merged**